kani-driver/src/call_cargo.rs

```diff
@@ -23,13 +23,13 @@ pub struct CargoOutputs {
 
 impl KaniSession {
     /// Calls `cargo_build` to generate `*.symtab.json` files in `target_dir`
-    pub fn cargo_build(&self) -> Result<CargoOutputs> {
+    pub fn cargo_build(&self, has_run_once: bool) -> Result<CargoOutputs> {
```

Comment:
I think instead of `has_run_once` you should either name it `build_gotoc` or even change this to take the the following enum parameter:
```
enum Backend {
  GOTOC,
  LLVM,
}
```

That will scale nicely once we add other backends. That said, I believe @zhassan-aws was suggesting that we use something different than backend. So we might want to ask if he has any other suggestion.